### DISCUSSION TITLE
M5 Atom - Ethernet and Reboot deviceID

### DISCUSSION THREAD

**Discussion by honey-pablo** [josephdadams/TallyArbiter#746] [Ideas]
Hi,

I'm using the M5 Atom Matrix controller with this [code](https://github.com/josephdadams/TallyArbiter/blob/master/listener_clients/M5AtomMatrix-listener/tallyarbiter-m5atom/tallyarbiter-m5atom.ino) 

- When rebooting the M5 Atom it creates a new listener entry in Tally Arbiter,  it appears to generate a new Device ID each boot - it does appear to retain it's association with which camera device it should be but is there a way to ensure the M5 Atom appears as one line and survives a reboot?

- I have the AtomPoE,  a plug in Ethernet module for the M5 Atom.  Does anyone have experience making this work with this code,  iv'e tried to remove the wifi config and replace with the ethernet examples from M5 but can't get it to work.  Any help would be very much appreciated as I'm using a broadcast IP mesh for transmission and can't rely on the standard wifi.

Cheers.

**Comment by Jocke4f** [comment]
The issue with "new" Device ID after rebooting M5 Atom was solved in https://github.com/josephdadams/TallyArbiter/pull/811.

I don't know how setup AtomPoE.
